x-pack/plugins/apm/public/application/csmApp.tsx

```diff
@@ -20,8 +20,8 @@ import { APMRouteDefinition } from '../application/routes';
 import { renderAsRedirectTo } from '../components/app/Main/route_config';
 import { ScrollToTopOnPathChange } from '../components/app/Main/ScrollToTopOnPathChange';
 import { RumHome, UX_LABEL } from '../components/app/RumDashboard/RumHome';
-import { ApmPluginContext } from '../context/ApmPluginContext';
-import { UrlParamsProvider } from '../context/UrlParamsContext';
+import { ApmPluginContext } from '../context/apm_plugin_context/apm_plugin_context';
+import { UrlParamsProvider } from '../context/url_params_context/url_params_context';
```

Comment:
This looks weird. I'd rather having something like:
```suggestion
import { UrlParamsProvider } from '../context/url_params_context';
```

Reply:
WDYT about 
```
import { UrlParamsProvider } from '../context/url_params/url_params_context'
```

Thing is, the folder contains both a hook and a context and I think each should have a clear filename to make them easier to recognise. In your example the context would be named `index.tsx`

Reply:
`import { UrlParamsProvider } from '../context/url_params/url_params_context'` looks good to me.